charts/redpanda/values.yaml

```diff
@@ -92,9 +92,23 @@ external:
   # External config doesn't apply to RPC listeners as they are never externally accessible
   # These values can be overridden by each listener if needed
   enabled: true
-  # Default external access type (NodePort is the only option for now)
+  # Default external access type (only NodePort at the moment)
   type: NodePort
+  # Domain to be used for each node
   domain: local
+  #
+  # nodeType can be set to the following: subdomain ip
+  # with subdomain, each node will advertise an address with <external.nodes[replica index]>.<domain>
+  # with ip, each node will advertise only <external.nodes[replica index]> (and domain will be ignored)
+  nodeType: ip
+  #
+  # The nodes list must have an entry for each statefulset replica
+  # By default, the statefulset is called redpanda and has 3 replicas
+  # If you want to access nodes by their IP: nodes: "18.224.215.250 18.118.163.26 3.142.153.29"
+  # Or if you want to access nodes via DNS: nodes: "apple bacon carrot"
+  # nodes: "apple bacon carrot"
+  nodes: "3.129.52.203 3.12.164.46 3.144.145.170"
```

Comment:
Can we use a term that's less ambiguous than "nodes"? I'm not sure we should be using routable addresses in the example.

How will a user know which ip addresses (or hostnames) to put here? Won't these be unpredictable?

Also, did you mean for this to be set to these addresses by default?

Reply:
Same as my `nodeType` comment above, this was mistakenly left in (I've removed it).